layout/_partial/head.ejs

```diff
@@ -4,6 +4,7 @@
         © Material Theme
         https://github.com/viosey/hexo-theme-material
         Version: 1.3.4 -->
+    <script>window.materialVersion = "1.3.4"</script>
```

Comment:
No, I and @EYHN had test, it works alright.
I think we need to consider about it.